src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -1071,11 +1079,11 @@ private async Task<T> ExecuteWithFailOverPolicyAsync<T>(
                     }
                 }
 
-                Uri currentEndpoint = _configClientManager.GetEndpointForClient(clientEnumerator.Current);
+                Uri currentEndpoint = _configClientManager.GetEndpointForClient(clientEnumerator.Current)!;
```

Comment:
`IConfigurationClientManager.GetEndpointForClient` can return `null` if it doesn't find the client, but everywhere it's used return value is never checked for null (here's an example)